com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/meta.properties

```diff
@@ -361,6 +361,9 @@ bundle_name.type = string
 bundle_name.help = bundle short name (CFBundleName). max 15 characters
 bundle_name.default =
 
+build_number.type = string
+build_number.help = build number (CFBundleVersion). numeric or x.y.z
```

Comment:
Given the "bundle_name"/CFBundleName correlation, I wonder if this should be called "bundle_version"?

Reply:
Yes, that makes sense I think.